script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt

```diff
@@ -21,30 +21,30 @@ She says firmly.
 The devil's words, merciless in content and chilling in tone, reverberate across the dining room.
 <Page7>
 "―――From whom, and what kind?”
-"For you Nii-san, I want you to have an appropriate attitude."
+"For you, Nii-san, and I want you to have an appropriate attitude."
 I knew it...
 So that's what she was thinking while eating, with her shapely eyebrows twitching...
 <Page8>
 "...What exactly do you mean by that?"
 "The first thing to do is to improve your lifestyle. No going out at night, of course. The curfew is set at eight o'clock.
-Whenever you go out for any reason, you have to ask permission from me, Kohaku or Hisui."
+Whenever you go out for any reason, you have to ask permission from me, Kohaku, or Hisui."
 <Page9>
 "Wha...?"
-Eight o'clock curfew? At this age? And before that, I need someone's permission to go outside?
-"Wait wait, that's..."
+Eight o'clock curfew? In this age? And before that, I need someone's permission to go outside?
+"Wait, wait, that's..."
 <Page10>
-"I will be setting up a study schedule on days off.
+"I will be setting up a study schedule for your days off.
 According to the reports, your grades are just about average, are they not?
-...#My goodness. You are only good at cutting corners efficiently as ever."
+...#My goodness. You are only good at cutting corners, efficiently as ever."
 "No way. I always do my best."
 <Page11>
 "But while you live in this mansion, I will have you take things seriously, so be prepared for that.
 I know you are a person who can do it if you put your mind to it, so even taking your health into account, you should be in the top three in your grade, should you not?"
 <Page12>
-It is a pain to be overestimated by your sister.
-I'm also afraid that she has already investigated my grades.
+It's a pain to be overestimated by your sister.
```

Comment:
**my** sister

Reply:
> 妹の過大評価が辛い

I get what you mean but I think he's being specific about this case rather than generalising.